modules/integration_azure-event-hub/conf/01-capacity-throttling.yaml

```diff
@@ -0,0 +1,32 @@
+
+---
+module: "Azure Event Hub"
+name: "throttled requests"
+filtering: "filter('resource_type', 'Microsoft.EventHub/clusters') and filter('primary_aggregation_type', 'true')"
+aggregation: ".mean(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
+transformation: ".min(over='15m')"
+value_unit: "%"
+signals:
+  A:
+    metric: ThrottledRequests
+  B:
+    metric: IncomingRequests
+  signal:
+    formula: "(A/B).scale(100).fill(0)"
+rules:
+  critical:
+    threshold: 50
+    comparator: ">"
+  major:
+    threshold: 30
+    comparator: ">"
+    dependency: critical
+  minor:
+    threshold: 10
+    comparator: ">"
+    dependency: major
+  warning:
```

Comment:
is it really useful / desirable to have such a lot of severities ?

the role of this repo is also to advice a relevant severity for each detector created following our https://github.com/claranet/terraform-signalfx-detectors/wiki/Notifications-binding.